mms_msg/sampling/pattern/meeting/scenario_sequence_sampler.py

```diff
@@ -145,3 +146,12 @@ def sample_asymmetric(scenarios, examples, rng: np.random.Generator, target_acti
     assert np.isclose(np.sum(p_scenario), 1)
 
     return rng.choice(scenarios, p=p_scenario)
+
+
+scenario_sequence_samplers = Dispatcher({
+    'random': sample_random,
+    'round_robin': sample_round_robin,
+    'random_round_robin': sample_random_round_robin,
+    'balanced': sample_balanced,
+    'balanced_no_repeat': sample_balanced_no_repeat,
```

Comment:
Is the asymmetric sampler still missing, here?

Reply:
You can't instantiate it with just a string because it needs additional parameters. So, yes, it is missing, but it is not possible to add it